rosidl_runtime_rs/src/sequence.rs

```diff
@@ -92,6 +92,16 @@ pub struct SequenceIterator<T: SequenceAlloc> {
 // ========================= impl for Sequence =========================
 
 impl<T: SequenceAlloc> Clone for Sequence<T> {
+    /// Creates a deep copy of the sequence.
+    ///
+    /// # Panics
```

Comment:
This `# Panics` and the `# Safety` sections are good adds! This is exactly what the rust book encourages us to do.
https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html